config.yaml

```diff
@@ -1,16 +1,76 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2023 Canonical Ltd.
```

Comment:
Yes, Juju is not handling correctly the the maximum value for a Java long (Juju does some weird rounding), so the only way to avoid this issue is to use strings.